README.mdown

```diff
@@ -166,6 +167,39 @@ supports the "groupPrefix" parameter. This will add a prefix to the Ganglia metr
 The Zabbix reporter requires the dependency `zabbix-sender-0.0.3.jar` (Maven group ID: io.github.hengyunabc,
 Maven artifact ID: zabbix-sender, version: 0.0.3).
 
+### InfluxDB
+
+The InfluxDB reporter currently requires the dependency [`metrics-influxdb-1.1.6.jar`](http://repo1.maven.org/maven2/com/izettle/metrics-influxdb/1.1.6/metrics-influxdb-1.1.6.jar)
+[Sources](https://github.com/iZettle/dropwizard-metrics-influxdb) (Maven group ID: com.izettle,
+Maven artifact ID: metrics-influxdb, version 1.1.6).
+
+Dropwizard 4.0 will introduce native InfluxDB support.
+
+InfluxDB connection requires a few more configuration properties:
+
+    influxdb:
+      -
+        period: 60
+        timeunit: 'SECONDS'
+        protocol: 'http'
+        auth: 'troll:admin123'
+        dbName: 'db1'
+        connectionTimeout: 1000
+        readTimeout: 1000
+        tags:
+        - host: 'node1'
```

Comment:
I think it might be better to show the `host.name` example value that will get set as the hostname. Also, we will need to make sure the influxdb docs on the README mention it has the same 4 value replace options as graphite/ganglia.